### ISSUE TITLE
Video level accuracy - small question

### ISSUE THREAD

**Issue by Esaada** [cypw/PyTorch-MFNet#28]
Hi, I'm trying to understand your video level accuracy calculations.
Generally, and please correct me if I'm wrong, you take a video  and sample N clips, accumulate the outputs, and then take the  top1, right? let's say we have 2 classes and the results are:
0.3 , 0.7
1    , 0
0.4, 0.6
The predicted label will be 1?
And now I'm trying to understand your calculations, I couldn't understand how come your multiplying part with 0.92 and a part with 0.08 on the test file.
I'll be happy for a short explanation.

Thanks!

**Comment by cypw** [comment]
1) Correct. The predicted label will be the first class.
2) 0.92 and 0.08 are weights for calculating the running mean for the progress bar. They do not affect the final accuracy and can be safely removed if you don't understand them.

**Comment by Esaada** [comment]
Ok , and just one more question  I'm trying to adjust your evaluation method but I'm getting much higher results on hmdb51 while sampling much less than 40 clips per video, and I'm trying to understand why.
any idea?

**Comment by cypw** [comment]
Is that because you are making an inaproperate comparison?
The 74.6% Top-1 accuracy on HMDB51 is an averaged accuracy on slip1, slip2, and slip3.
While, the released `MFNet3D_HMDB51_Split-1_75.8.pth` is only for HMDB51 (split-1), which got 75.8% Top-1 accuracy.

**Comment by Esaada** [comment]
I evaluated on the validation set :( sorry for bothering
